parser/src/cfg/defcfg.rs

```diff
@@ -348,6 +364,41 @@ pub fn parse_colon_separated_text(paths: &str) -> Vec<String> {
     all_paths
 }
 
+#[cfg(target_os = "macos")]
+pub fn parse_macos_dev(val: &SExpr) -> Result<Vec<String>> {
```

Comment:
`parse_linux_dev` does literally the same thing, so this one should be removed and `parse_linux_dev` renamed to something that makes sense for both linux and macOS eg `parse_devices`.